src/content/en/updates/2019/07/image-support-for-async-clipboard.md

```diff
@@ -112,37 +116,40 @@ permissionStatus.onchange = () => {
 };
 ```
 
-## 🆕 The new image-focused portion of the Asynchronous Clipboard API {: #images }
+## The new image-focused portion of the Asynchronous Clipboard API {: #images }
 
-### Copy: Writing an image to the clipboard {: #copy-image }
+### Copy: writing an image to the clipboard {: #copy-image }
 
 The new `navigator.clipboard.write()` method can be used for copying images
 to the clipboard. Like [`writeText()`](#copy-text), it is asynchronous, and
 Promise-based. Actually, `writeText()` is just a convenience method for the
 generic `write()` method.
 
-In order to write an image to the clipboard, you need the image as a
-[`Blob`](https://developer.mozilla.org/en-US/docs/Web/API/Blob). One way to
-achieve this is by `fetch`ing (or `XMLHttpRequest`ing) the image from a
-server and getting the response body as a [Blob][blob] (or for XHR,
-by setting the [`responseType`][blog-response-type] to `'blob'`).
-Another method to `Blob`ify an image is to write the image to a canvas, then
-call the `canvas`’s [`toBlob()`][to-blob] method.
+To write an image to the clipboard, you need the image as a
+[`Blob`][blob]. One way to do
+this is by requesting the image from an server by calling `fetch()` (or
+`XMLHTTPReuest()`). The `response` object returned by `fetch()` has a [`blob()`
+method][blob-method] and
+`XMLHTTPRequest()` let's you set `"blob"` as the [`responseType`][blog-response-type]. 
+
+Calling the server may not be desireable or possible for a variety of reasons.
+Fortunately, you can also write the image to a canvas, then call
+[`HTMLCanvasElement.toBlog()`][to-blob].
 
-[blob]: https://developer.mozilla.org/en-US/docs/Web/API/Body/blob
+[blob]: https://developer.mozilla.org/en-US/docs/Web/API/blob
+[blob-method]: https://developer.mozilla.org/en-US/docs/Web/API/Body/blob
 [to-blob]: https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob
 [blog-response-type]: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType#Value
 
-Next, pass an array of `ClipboardItem`s as a parameter to the `write()` method.
+Next, pass an array of `ClipboardItem` objects as a parameter to the `write()` method.
 Currently you can only pass one image at a time, but we plan to add support for
 multiple images in the future.
 
 The `ClipboardItem` takes an object with the MIME type of the image as the key,
 and the actual blob as the value. The sample code below shows a future-proof
-way to do this by leveraging the [`Object.defineProperty()`][object-define-prop]
-method. Using this approach will ensure your code will be ready for future
-image types as well as other MIME types that the Asynchronous Clipboard API
-may support.
+way to do this by using the [`Object.defineProperty()`][object-define-prop]
```

Comment:
'leveraging' to 'using'

'Leveraging' means 'to force'. You're not forcing anything here. You're simply using it.

Reply:
TIL this subtlety. I was leveraging the word leverage all wrong for my entire life. Pun intended.  Thanks for teaching me English!